tutorials/tut-hash-distribution.rst

```diff
@@ -182,7 +182,7 @@ bots:
 
 Above, when we created our two tables, we partitioned them along the same
 column and created an equal number of shards for each. Doing this means that
-all data for each editor is kept on the same machine, or, colocated.
+all data for each editor is kept on the same machine, or, co-located.
```

Comment:
Add a link here to our description? We get a fair bit of traffic to our tutorial section, so we can direct some of that to this content.